### PR TITLE
Updated readme

### LINKED ISSUE #85: Update the README
Include:

- instructions about how to publish to our github host: https://onja-madagascar.github.io/music-of-madagascar/
- what you need to get started
- how to add a new artist
- and why this project started

### PR DESCRIPTION
**Pull request by ndorfin**
Fixes #85 

Makes the README more user-friendly, and includes instructions on how to update the website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+# Music of Madagascar
 
+A list of artists and genres from Madagascar.
+See: [onja-madagascar.github.io/music-of-madagascar](https://onja-madagascar.github.io/music-of-madagascar/)
+
+## About
+
+This repo was started as a school project at [Onja.org](https://onja.org), as we love Malagasy music.
+
+## How to contribute
```
Comment (vakodrazan) [inline README.md:10]: It is not that clear so you should make some detail in how to contribute.

**Review by vakodrazan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ndorfin) [reply README.md:10]: Ah, my headings look wrong. One can contribute by developing, or adding artists, or adding genres

Comment (ndorfin) [reply README.md:10]: Thanks, will fix the heading for development

**Review by vakodrazan — APPROVED** [review]
You got it. Cool!

**Pull request merged**